pages/share.jsx

```diff
@@ -2,9 +2,13 @@ import React  from 'react';
 import Footer from '../components/footer.jsx';
 import Header from '../components/header.jsx';
 import Social from '../components/social.jsx';
+import analytics from '../public/js/analytics.js';
 
 var ThankYou = React.createClass({
   mixins: [require('react-intl').IntlMixin],
+  componentDidMount: function() {
+    analytics();
```

Comment:
Based on this https://github.com/mozilla/donate.mozilla.org/issues/103 not sure if I misunderstood something from there?